picard/config.py

```diff
@@ -195,6 +195,10 @@ def register_upgrade_hook(self, func, *args):
 
     def run_upgrade_hooks(self, outputfunc=None):
         """Executes registered functions to upgrade config version to the latest"""
+        if self._version == Version(0, 0, 0, 'dev', 0):
+            # This is a freshly created config
+            self._version = PICARD_VERSION
+            self._write_version()
```

Comment:
Perhaps this should go in a separated PR, it seems unrelated (though it makes sense)

Reply:
True, I added https://github.com/metabrainz/picard/pull/1733 . I will rebase this PR once that got merged.